plugins/modules/ec2_instance.py

```diff
@@ -1699,6 +1751,51 @@ def handle_existing(existing_matches, state):
     return result
 
 
+def enforce_count(existing_matches, module):
+    exact_count = module.params.get('exact_count')
+    changed = False
+
+    try:
+        instances = existing_matches
```

Comment:
```suggestion
        current_count = len(existing_matches)
```
`instances` is only used once in the following code; the length is much more frequently utilized.  Setting a variable to the length one time rather than copying the variable and calling len() on it each time will make the following a little more descriptive and easier to follow.